ops/jenkins/bfd-mgmt-synthea-generation-build/build-push.sh

```diff
@@ -64,33 +74,32 @@ download_synthea_properties() {
 }
 
 download_synthea_latest_jar() {
-  echo "Downloading latest Synthea JAR to $BUILD_CONTEXT_ROOT_DIR"
-  curl -LkSs "${SYNTHEA_LATEST_JAR_URL}" -o "$BUILD_CONTEXT_ROOT_DIR/${SYNTHEA_JAR_FILE}"
+  echo "Downloading latest Synthea JAR to $CONTEXT_DIR"
+  curl -LkSs "${SYNTHEA_LATEST_JAR_URL}" -o "$CONTEXT_DIR/${SYNTHEA_JAR_FILE}"
 }
```

Comment:
We want these files in the docker image...right? Was it correct to update their dest?

Reply:
Yep, we need the build Synthea `.jar` in the image otherwise there's nothing to run. As far as I understand this, this is the correct approach.